features/s3/postObject.feature

```diff
@@ -0,0 +1,18 @@
+@s3 @integ
+Feature: POST object uploads
+
+    Scenario: Upload an object via POST object with Signature V2
+        Given I have an s3 client and I have a file
+        When I create a POST object in SigV2 with:
```

Comment:
The V2 signer just takes a string. Would it make more sense to use a [pyString](http://docs.behat.org/en/v3.0/guides/1.gherkin.html#pystrings) instead of a table?